grommet/grommet
# Removed error from Carousel test

#### What does this PR do?
Replaces the call to `setTimeout()` in `Carousel-test.js` with mocked timers, and wraps change in `act()` to remove warning from test. 
#### Where should the reviewer start?
`grommet/src/js/components/Carousel/__tests__/Carousel-test.js` line 66 ('play' test)
#### What testing has been done on this PR?
I have ran `yarn test Carousel-test` to ensure that this PR will remove the error from the test
#### How should this be manually tested?
Run `yarn test Carousel-test` to ensure that the tests still work and the error was resolved
#### What are the relevant issues?
#4177 
#### Screenshots (if appropriate)
<img width="946" alt="Screen Shot 2020-06-17 at 11 24 16 AM" src="https://user-images.githubusercontent.com/49867133/84918073-2755cb00-b08e-11ea-82f3-356175db8312.png">

#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
Backwards compatible
